ivy/ivy.xml

```diff
@@ -78,7 +78,11 @@
                 <dependency org="org.apache.cxf" name="cxf-rt-transports-http-jetty" rev="3.0.4"/>
                 <dependency org="com.fasterxml.jackson.core" name="jackson-databind" rev="2.5.1" /> 
                 <dependency org="com.fasterxml.jackson.dataformat" name="jackson-dataformat-cbor" rev="2.5.1" />
-                <dependency org="com.fasterxml.jackson.jaxrs" name="jackson-jaxrs-json-provider" rev="2.5.1" />	
+                <dependency org="com.fasterxml.jackson.jaxrs" name="jackson-jaxrs-json-provider" rev="2.5.1" />
+                <dependency org="org.apache.mahout" name="mahout-math" rev="0.8" />
```

Comment:
The Mahout and Lucene dependencies cannot be included in main ivy.xml if this is to be a plugin.